.github/workflows/dispatch-rebottle.yml

```diff
@@ -29,27 +29,113 @@ env:
   HOMEBREW_RELOCATE_RPATHS: 1
 
 jobs:
+  setup:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/homebrew/ubuntu16.04:master
+    outputs:
+      runners: ${{steps.determine-runners.outputs.runners}}
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Determine runners
+        id: determine-runners
+        uses: Homebrew/actions/brew-script@master
+        env:
+          HOMEBREW_FORMULA_TO_REBOTTLE: ${{github.event.inputs.formula}}
+          HOMEBREW_REBOTTLE_TIMEOUT: ${{github.event.inputs.timeout}}
+        with:
+          script: |
```

Comment:
I wonder if it makes more sense to dump this script in `.github/scripts` or something. That way editors can parse this as Ruby; maybe that will make it easier to modify later on.

Reply:
Yeah that makes sense. `Homebrew/actions/brew-script` doesn't currently support doing so but I guess at this point we could just do `brew ruby` directly anyway.